components/calendar/src/iso.rs

```diff
@@ -276,6 +276,80 @@ impl DateTime<Iso> {
             time: types::Time::try_new(hour, minute, second, 0)?,
         })
     }
+
+    /// Minute count representation of calendars starting from 00:00:00 on Jan 1st, 1970.
+    ///
+    /// ```rust
+    /// use icu::calendar::DateTime;
+    ///
+    /// let today = DateTime::new_iso_datetime(2020, 2, 29, 0, 0, 0).unwrap();
+    ///
+    /// assert_eq!(today.minutes_since_local_unix_epoch(), 26382240);
+    /// assert_eq!(
+    ///             DateTime::from_minutes_since_local_unix_epoch(26382240),
+    ///             Ok(today)
+    ///         );
+    ///
+    /// let today = DateTime::new_iso_datetime(1970, 1, 1, 0, 0, 0).unwrap();
+    ///
+    /// assert_eq!(today.minutes_since_local_unix_epoch(), 0);
+    /// assert_eq!(DateTime::from_minutes_since_local_unix_epoch(0), Ok(today));
+    /// ```
+    pub fn minutes_since_local_unix_epoch(&self) -> i32 {
```

Comment:
Thought: We need to think about what happens when overflow occurs, but this is a problem that is all over the icu_calendar crate, not just this function, so we shouldn't block this PR on a resolution to it. I opened https://github.com/unicode-org/icu4x/issues/2151

Reply:
sg!